services/productService.js

```diff
@@ -0,0 +1,16 @@
+const productDao = require('../models/productDao');
+const userDao = require('../models/userDao');
+const appError = require('../middlewares/appError');
+
+const createReview = async (content, userId, productId) => {
+    const user = await userDao.getUserById(userId);
+    const product = await productDao.getProductById(productId);
+
+    if (!user || !product) throw new appError('DATA_EXIST', 409);
+
+    await productDao.createReview(content, userId, productId);
+}
```

Comment:
위에서 드린 피드백에 이어서, 이미 userId는 middleware에서 존재 유무를 확인했기 때문에 한 번 더 조회하는 것은 불필요합니다.

```suggestion
const createReview = async (content, userId, productId) => {
    const product = await productDao.getProductById(productId);

    if (!user || !product) throw new appError('DATA_EXIST', 409);

    await productDao.createReview(content, userId, productId);
}
```